### ISSUE TITLE
Hub is unknown to this Hublet

### ISSUE THREAD

**Issue by kcleong** [HubSpot/hubspot-api-python#261]
I first posted this on the community forum (https://community.hubspot.com/t5/APIs-Integrations/Hub-is-unknown-to-this-Hublet/m-p/916937#M70825) but I order to reach out to the devs of this library I am also posting my issue/question here.

For a customer I am using the Hubspot python API ([https://github.com/HubSpot/hubspot-api-python/](https://github.com/HubSpot/hubspot-api-python/tree/master)) with an access token / private app.

Intermittently (as in the last weekend and before that on Jan 8th) we are getting these errors:

```
{
  "status":"error",
  "message":"Hub 5604529 is unknown to this Hublet, but it appears to exist in Hublet eu1",
  "correlationId":"1a6d0728-d441-42e9-a87e-ae225b49bd33",
  "correctHublet":"eu1"
}
```

Please mind, most of the time the API integration is working correctly but it looks like once a month these errors are popping up.

I know the Hubspot data was moved last year (I believe in December) from the US to the EU. However I don't see any option in the hubspot-api-python to point to a specific geograpical region. And most URL's seem to be hardcoded in hubspot-api-python.

**Comment by altugsogutoglu** [comment]
Did you fix this issue? Im having the same problems

**Comment by minhtri0795** [comment]
Have any progress here, I have same problems ?

**Comment by altugsogutoglu** [comment]
@minhtri0795 yes, i created a new api key (or you can refresh a old one). And you will see a key with eu instead of na. When you see this the api should work fine.

**Comment by alzheltkovskiy-hubspot** [comment]
Hey all. Create a new API key (or refresh an old one) – this should help. Please let us know if it solves your issue.
